### PR TITLE
Wasteplanet Pandora Remap (REQUIRES #1945)

### PR DESCRIPTION
**Pull request by Zevotech**
## About The Pull Request
Remaps wasteplanet_pandora to not be a tiny box with a one-way entry into a locked room with a pandora. Requires #1945 to be merged or at least test merged.

image

## Why It's Good For The Game
shit ruin must remap

## Changelog

:cl:
Add: Remapped wasteplanet_pandora
/:cl:

### CODE REVIEW

**Comment by Zevotech** [comment]
is there genuinely a fucking check dedicated to the spelling of centcom? im dying.

**Comment by Valorium-RB** [comment]
The note(s) used are a bit too brazen, methinks. If it were wrapped in about three pounds of corpo-speak and less overtly colonialist then it'd be about fine.

**Comment by Zevotech** [comment]
> The note(s) used are a bit too brazen, methinks. If it were wrapped in about three pounds of corpo-speak and less overtly colonialist then it'd be about fine.

its meant to be old "practically a government" nanotrasen so thats what i went for. examples of how to improve it are appreciated though.

**Comment by Latentish** [comment]
> > The note(s) used are a bit too brazen, methinks. If it were wrapped in about three pounds of corpo-speak and less overtly colonialist then it'd be about fine.
> 
> its meant to be old "practically a government" nanotrasen so thats what i went for. examples of how to improve it are appreciated though.

Remove the colonial racism, make the note less edgy.
Or remove it entirely. It doesn't totally need to be there.

**Comment by Zevotech** [comment]
> > > The note(s) used are a bit too brazen, methinks. If it were wrapped in about three pounds of corpo-speak and less overtly colonialist then it'd be about fine.
> > 
> > 
> > its meant to be old "practically a government" nanotrasen so thats what i went for. examples of how to improve it are appreciated though.
> 
> Remove the colonial racism, make the note less edgy. Or remove it entirely. It doesn't totally need to be there.

yk what good point. entirely environmental storytelling it is.

**Comment by Zevotech** [comment]
up to date and tm ready

**Comment by Erikafox** [comment]
resolve your merge conflicts

**Comment by PrefabQuasar** [comment]
looks like a large improvement over the original, once merge conflicts are fixed, id like to see it TM'd

**Comment by Zevotech** [comment]
that should handle the conflicts. not sure why github is inserting the file comparison marks directly into the code but at least its easily fixed

**Comment by Latentish** [comment]
Please update your images.
Also I agree with prefab, tm'able.

**Review by Latentish — APPROVED** [review]
I like this. I tested it in game, it's pretty good. I like the arena.

File: code/datums/mapgen/planetary/WasteGenerator.dm [hunk 1]
```diff
@@ -89,6 +89,7 @@
 /datum/biome/waste
 	open_turf_types = list(
 		/turf/open/floor/plating/asteroid/wasteplanet/lit = 80,
+		/turf/open/floor/plating/rust/wasteplanet/lit = 15,
```
Comment (Latentish) [inline code/datums/mapgen/planetary/WasteGenerator.dm]: Why are you adding this in? It seems to be defined below already.

**Review by Latentish — CHANGES_REQUESTED** [review]

File: code/game/turfs/open/floor/plating/wasteplanet.dm [hunk 2]
```diff
@@ -48,4 +53,45 @@
...
+	light_color = LIGHT_COLOR_LAVA
+
+/turf/open/water/waste
+	initial_gas_mix = OPENTURF_DEFAULT_ATMOS
+	planetary_atmos = TRUE
+
+/turf/open/water/waste/lit
+	light_range = 2
+	light_power = 0.2
+	light_color = LIGHT_COLOR_LAVA
```
Comment (Erikafox) [inline code/game/turfs/open/floor/plating/wasteplanet.dm]:
```suggestion
	light_color = LIGHT_COLOR_FLARE
```

File: code/game/turfs/open/floor/plating/wasteplanet.dm [hunk 3]
```diff
@@ -48,4 +53,45 @@
...
+
+/turf/open/floor/plating/dirt/old/waste
+	initial_gas_mix = OPENTURF_DEFAULT_ATMOS
+	planetary_atmos = TRUE
+
+/turf/open/floor/plating/dirt/old/waste/lit
+	light_range = 2
+	light_power = 0.2
+	light_color = LIGHT_COLOR_LAVA
```
Comment (Erikafox) [inline code/game/turfs/open/floor/plating/wasteplanet.dm]:
```suggestion
	light_color = LIGHT_COLOR_FLARE

File: code/game/turfs/open/floor/plating/wasteplanet.dm [hunk 4]
```diff
@@ -48,4 +53,45 @@
...
+
+/turf/open/floor/plating/grass/wasteplanet
+	icon_state = "junglegrass"
+	initial_gas_mix = OPENTURF_DEFAULT_ATMOS
+	planetary_atmos = TRUE
+
+/turf/open/floor/plating/grass/wasteplanet/lit
+	light_range = 2
+	light_power = 0.2
+	light_color = LIGHT_COLOR_LAVA
```
Comment (Erikafox) [inline code/game/turfs/open/floor/plating/wasteplanet.dm]:
```suggestion
	light_color = LIGHT_COLOR_FLARE
```

File: code/game/turfs/open/floor/plating/wasteplanet.dm [hunk 5]
```diff
@@ -48,4 +53,45 @@
 /turf/open/water/tar/waste/lit
 	light_range = 2
 	light_power = 0.2
+	light_color = LIGHT_COLOR_LAVA
+
+/turf/open/water/waste
+	initial_gas_mix = OPENTURF_DEFAULT_ATMOS
+	planetary_atmos = TRUE
+
+/turf/open/water/waste/lit
```
Comment (Erikafox) [inline code/game/turfs/open/floor/plating/wasteplanet.dm:84]: define the lits where the rest of them are defined in file. That is to say at the bottom.

File: code/game/turfs/open/floor/plating/wasteplanet.dm [hunk 6]
```diff
@@ -48,4 +53,45 @@
 /turf/open/water/tar/waste/lit
 	light_range = 2
 	light_power = 0.2
+	light_color = LIGHT_COLOR_LAVA
```
Comment (Erikafox) [inline code/game/turfs/open/floor/plating/wasteplanet.dm]:
```suggestion
	light_color = LIGHT_COLOR_FLARE
```

**Review by Erikafox — CHANGES_REQUESTED** [review]

File: code/game/turfs/open/floor/plating/wasteplanet.dm [hunk 7]
```diff
@@ -48,4 +53,45 @@
...
+	initial_gas_mix = OPENTURF_DEFAULT_ATMOS
+	planetary_atmos = TRUE
+
+/turf/open/water/waste/lit
+	light_range = 2
+	light_power = 0.2
+	light_color = LIGHT_COLOR_LAVA
+
+/turf/open/floor/plating/dirt/old/waste
+	initial_gas_mix = OPENTURF_DEFAULT_ATMOS
```
Comment (Erikafox) [inline code/game/turfs/open/floor/plating/wasteplanet.dm]: also:
```suggestion
	initial_gas_mix = WASTEPLANET_DEFAULT_ATMOS
```

File: code/game/turfs/open/floor/plating/wasteplanet.dm [hunk 8]
```diff
@@ -48,4 +53,45 @@
...
+	planetary_atmos = TRUE
+
+/turf/open/floor/plating/grass/wasteplanet/lit
+	light_range = 2
+	light_power = 0.2
+	light_color = LIGHT_COLOR_LAVA
+
+/turf/open/floor/wood/waste
+	initial_gas_mix = OPENTURF_DEFAULT_ATMOS
```
Comment (Erikafox) [inline code/game/turfs/open/floor/plating/wasteplanet.dm]:
```suggestion
	initial_gas_mix = WASTEPLANET_DEFAULT_ATMOS
```

File: code/game/turfs/open/floor/plating/wasteplanet.dm [hunk 9]
```diff
@@ -48,4 +53,45 @@
...
+	planetary_atmos = TRUE
+
+/turf/open/indestructible/hierophant/waste
+	initial_gas_mix = OPENTURF_DEFAULT_ATMOS
```
Comment (Erikafox) [inline code/game/turfs/open/floor/plating/wasteplanet.dm]:
```suggestion
	initial_gas_mix = WASTEPLANET_DEFAULT_ATMOS
```

File: code/game/turfs/open/floor/plating/wasteplanet.dm [hunk 10]
```diff
@@ -48,4 +53,45 @@
...
+	planetary_atmos = TRUE
+
+/turf/open/indestructible/hierophant/two/waste
+	initial_gas_mix = OPENTURF_DEFAULT_ATMOS
```
Comment (Erikafox) [inline code/game/turfs/open/floor/plating/wasteplanet.dm]:
```suggestion
	initial_gas_mix = WASTEPLANET_DEFAULT_ATMOS
```

**Review by Erikafox — CHANGES_REQUESTED** [review]
If something has planetary atmos it needs to match the atmosphere of the planet it's on, otherwise you're just making atmos lag fnr.

File: code/datums/mapgen/planetary/WasteGenerator.dm (hunk 1 above)
Comment (Zevotech) [reply code/datums/mapgen/planetary/WasteGenerator.dm]: I don't think I added that. This PR was made before the biomes were merged so any pulls from that probably got dragged into this.

Comment (Erikafox) [reply code/datums/mapgen/planetary/WasteGenerator.dm]: It's probably left over but it does need to go.

Comment (Zevotech) [reply code/datums/mapgen/planetary/WasteGenerator.dm]: Fucky Wucky Obliterated !

**Comment by Zevotech** [comment]
images updated and reviews addressed

File: code/game/turfs/open/floor/plating/wasteplanet.dm [hunk 11]
```diff
@@ -28,13 +32,40 @@
...
+	planetary_atmos = TRUE
+
 //open turfs then open lits.
 
 /turf/open/floor/plating/wasteplanet/lit
 	light_range = 2
 	light_power = 0.2
 	light_color = LIGHT_COLOR_FLARE
 
+/turf/open/floor/plating/rust/wasteplanet/lit
```
Comment (Erikafox) [inline code/game/turfs/open/floor/plating/wasteplanet.dm:68]: you need to scrub this out. the already defined path is /plating/wasteplanet/rust/lit

File: code/datums/mapgen/planetary/WasteGenerator.dm [hunk 12]
```diff
@@ -368,6 +374,7 @@
 
 /datum/biome/cave/waste/metal //deeper in, there's no normal stuff here
 	open_turf_types = list(
+		/turf/open/floor/plating/rust/wasteplanet = 10,
```
Comment (Erikafox) [inline code/datums/mapgen/planetary/WasteGenerator.dm:377]: here

File: code/datums/mapgen/planetary/WasteGenerator.dm [hunk 13]
```diff
@@ -334,6 +339,7 @@
 /datum/biome/cave/waste/tar_bed //tar colorings here
 	open_turf_types = list(
 		/turf/open/floor/plating/asteroid/wasteplanet = 70,
+		/turf/open/floor/plating/rust/wasteplanet = 20,
```
Comment (Erikafox) [inline code/datums/mapgen/planetary/WasteGenerator.dm:342]: here

File: code/datums/mapgen/planetary/WasteGenerator.dm [hunk 14]
```diff
@@ -274,6 +278,7 @@
 /datum/biome/cave/waste
 	open_turf_types = list(
 		/turf/open/floor/plating/asteroid/wasteplanet = 80,
+		/turf/open/floor/plating/rust/wasteplanet = 15,
```
Comment (Erikafox) [inline code/datums/mapgen/planetary/WasteGenerator.dm:281]: here

File: code/datums/mapgen/planetary/WasteGenerator.dm [hunk 15]
```diff
@@ -265,6 +268,7 @@
 /datum/biome/waste/metal/rust
 	open_turf_types = list(
 		/turf/open/floor/plating/asteroid/wasteplanet/lit = 1,
+		/turf/open/floor/plating/rust/wasteplanet/lit = 10,
```
Comment (Erikafox) [inline code/datums/mapgen/planetary/WasteGenerator.dm:271]: here

File: code/datums/mapgen/planetary/WasteGenerator.dm [hunk 16]
```diff
@@ -220,6 +222,7 @@
 /datum/biome/waste/metal
 	open_turf_types = list(
 		/turf/open/floor/plating/asteroid/wasteplanet/lit = 5,
+		/turf/open/floor/plating/rust/wasteplanet/lit = 45,
```
Comment (Erikafox) [inline code/datums/mapgen/planetary/WasteGenerator.dm:225]: here

File: code/datums/mapgen/planetary/WasteGenerator.dm [hunk 17]
```diff
@@ -208,6 +209,7 @@
 /datum/biome/waste/tar_bed //tar colorings
 	open_turf_types = list(
 		/turf/open/floor/plating/asteroid/wasteplanet/lit = 70,
+		/turf/open/floor/plating/rust/wasteplanet/lit = 10,
```
Comment (Erikafox) [inline code/datums/mapgen/planetary/WasteGenerator.dm:212]: here

File: code/datums/mapgen/planetary/WasteGenerator.dm [hunk 18]
```diff
@@ -154,6 +154,7 @@
 /datum/biome/waste/crater
 	open_turf_types = list(
 		/turf/open/floor/plating/asteroid/wasteplanet/lit = 90,
+		/turf/open/floor/plating/rust/wasteplanet/lit = 10,
```
Comment (Erikafox) [inline code/datums/mapgen/planetary/WasteGenerator.dm:157]: here

**Review by Erikafox — CHANGES_REQUESTED** [review]

**Comment by Erikafox** [comment]
ping us when you feel like addressing the review I left.

**Comment by Zevotech** [comment]
maptainers reopen pls

**Pull request closed without merging**